### PR TITLE
Vogue IT example article.

### PR DESCRIPTION
**Pull request by nigelhanlon**
#### This resolves Clubhouse issue: https://app.clubhouse.io/condenastinternational/story/10302/add-vit-article-to-demo-api

- [ ] Tests have been written
- [x] All TODOs have a task created and assigned
- [x] Any tech debt raised as cards in Clubhouse
- [x] Checked that the new code runs properly

### CODE REVIEW

**Comment by thekemkid** [comment]
You probably want an example with inline entities/images.

**Comment by nigelhanlon** [comment]
Unfortunately the article I picked didn't have any images. I can add the main photo into it again I suppose as an inline image.

**Comment by nigelhanlon** [comment]
Changed category to pass with v0.7.0 of schema validator as well as bringing up to date with master. Added inline example image of the Stranger Things logo to the bottom of the article.

File: src/data/articles-vit.json [hunk 1]
```diff
@@ -1 +1,75 @@
...
+        "credit": "Getty Images",
+        "title": "Maya la figlia di Uma Thurman è nel cast di Stranger Things 3",
+        "filename": "GettyImages-927438610la.jpg"
+      }
+    ],
+    "categories": {
+      "sections": [
+        {
+          "name": "News",
+          "slug": "news",
```
Comment (srilq) [inline src/data/articles-vit.json]: I think we need to use the `moda` category. Ideally, we should use a `moda` article, but we could just use a news article and use the `moda` category for demonstrative purposes.

**Pull request merged**